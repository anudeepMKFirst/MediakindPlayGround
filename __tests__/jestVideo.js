import React from 'react';
import VideoPlayer from "./../video.youi.js";
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const vid = {title: "test"};
  global.navigation.getParam = jest.fn().mockImplementation(function (a) { return vid; });

  const tree = renderer
    .create(<VideoPlayer navigation={global.navigation}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});