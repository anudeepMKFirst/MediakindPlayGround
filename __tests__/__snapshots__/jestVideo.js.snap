// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
    }
  }
>
  <Composition
    onTimelinesLoaded={[Function]}
    source="Player_VideoRef"
  >
    <VideoRef
      metadata={Object {}}
      name="Video-Surface-View"
      onCurrentTimeUpdated={[Function]}
      onDurationChanged={[Function]}
      onPlaybackComplete={[Function]}
      onReady={[Function]}
      source={
        Object {
          "type": "HLS",
          "uri": "http://link.theplatform.com/s/BpkrRC/ckSTzzdGO_K3",
        }
      }
      src={
        Object {
          "drmInfo": undefined,
          "drmScheme": undefined,
          "isAsset": false,
          "isNetwork": true,
          "mainVer": 0,
          "patchVer": 0,
          "type": "HLS",
          "uri": "http://link.theplatform.com/s/BpkrRC/ckSTzzdGO_K3",
        }
      }
      style={
        Array [
          Object {
            "overflow": "hidden",
          },
          Object {
            "height": 1080,
            "position": "absolute",
            "width": 1920,
          },
        ]
      }
      testID="VideoSurface"
    />
  </Composition>
</View>
`;
